content/applications/general/payment_acquirers/stripe.rst

```diff
@@ -16,21 +16,47 @@ Credentials tab
 
 Odoo needs your **API Credentials** to connect with your Stripe account, which comprise:
 
-- Publishable Key: The key solely used to identify the account with Stripe.
-- Secret Key: The key to sign the merchant account with Stripe.
-- Webhook Signing Secret: If a webhook is enabled on your Stripe account
-  (:menuselection:`Developers --> webhooks`), this signing secret must be set to authenticate the
-  messages sent from Stripe to Odoo.
-
-To retrieve the publishable and secret keys, log into your Stripe dashboard and go to
-:menuselection:`Developers --> API Keys --> Standard Keys`
+- :ref:`Publishable Key <stripe/api_keys>`: The key solely used to identify the account with Stripe.
+- :ref:`Secret Key <stripe/api_keys>`: The key to sign the merchant account with Stripe.
+- :ref:`Webhook Signing Secret <stripe/webhook>`: When you enabled your webhook on your Stripe
+  account, this signing secret must be set to authenticate the messages sent from Stripe to Odoo.
 
 .. important::
    If you are trying Stripe as a test, in the **test mode**, change the **State** to *Test
    Mode*. We recommend doing this on a test Odoo database, rather than on your main database.
 
+.. _stripe/api_keys:
+
+Publishable and Secret keys
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To retrieve the publishable and secret keys, follow this `link to your api keys
+<https://dashboard.stripe.com/account/apikeys>`_, or log into your Stripe dashboard and go to
+:menuselection:`Developers --> API Keys --> Standard Keys`.
+
+.. _stripe/webhook:
+
+Webhook Signing Secret
+~~~~~~~~~~~~~~~~~~~~~~
+
+To retrieve the webhook signing secret, you'll first need to create this webhook. To do so, follow
```

Comment:
```suggestion
To retrieve the webhook signing secret, you first need to create a webhook. To do so, follow
```

Reply:
Indeed.

The present simple is the correct grammatical tense for general truths, things that always happen. The future is only for events that happen in the future, relative to the context. (for example here "to do this, you first need to create a webhook. After two weeks you will have to create a new one.")

See https://www.odoo.com/documentation/14.0/contributing/documentation/content_guidelines.html#grammatical-tenses